modules/rosetta/src/main/scala/org/tessellation/rosetta/api/generated/Operation.scala

```diff
@@ -5,9 +5,9 @@
   * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech). https://openapi-generator.tech
   */
 
-package org.tessellation.rosetta.server.model
+package org.tessellation.rosetta.api.generated
 
-import org.tessellation.rosetta.server.model.dag.metadataSchema.GenericMetadata
+import org.tessellation.rosetta.api.generated.dag.metadataSchema.GenericMetadata
```

Comment:
is `dag` also auto generated?

Reply:
The stuff in `org.tessellation.rosetta.api.generated.dag` is not autogenerated, but it is used with the auto-generated classes in `org.tessellation.rosetta.api.generated`. For example, it has enums.

Reply:
I moved the `dag` package to `org.tessellation.rosetta.api` and out of `org.tessellation.rosetta.api.generated`.

Reply:
Cool. You said that `api.generated` contains auto-generated classes so let's keep that exclusively for generated classes.